index.html

```
@@ -1,86 +1,141 @@
+        </div>
+      </div>
+    </section>
+  </main>
+
+  <div class="detail-backdrop" id="detail-backdrop" hidden></div>
+  <aside class="detail-panel" id="detail-panel" role="dialog" aria-modal="true" aria-labelledby="detail-title" hidden>
+    <button class="detail-close icon-button" id="detail-close" type="button" aria-label="Close details">
+      <svg viewBox="0 0 24 24" aria-hidden="true"><path d="M6 6l12 12M18 6 6 18"></path></svg>
+    </button>
+    <div class="detail-media" id="detail-media" hidden>
+      <img id="detail-image" src="" alt="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)